wpilibc/src/main/native/cpp/DriverStation.cpp

```diff
@@ -208,7 +208,6 @@ double DriverStation::GetStickAxis(int stick, int axis) {
   if (axis >= m_joystickAxes[stick].count) {
     // Unlock early so error printing isn't locked.
     m_cacheDataMutex.unlock();
-    lock.release();
```

Comment:
Shouldn't we be calling `lock.release()` rather then unlocking the mutex directly? Is the implicit `lock.release` called during the destructor smart enough to check if the lock is actually locked before unlocking?

Reply:
I was thinking of `lock.unlock()` here. `lock.release()` disassociates the mutex without unlocking it. `lock.unlock()` unlocks the mutex and it's smart enough to avoid double-unlocking when exiting the scope. I just changed it to use `lock.unlock()` instead of a manual mutex unlock.